oss-cloud-backend/table_definitions.sql

```diff
@@ -7,17 +7,21 @@ CREATE TABLE `contributions` (
   `dateCreated` datetime DEFAULT NULL,
   `status` varchar(7) DEFAULT NULL,
   `author` varchar(40) DEFAULT NULL,
-  PRIMARY KEY (`repo`,`owner`,`number`),
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  PRIMARY KEY (`id`),
   KEY `owner_FK_idx` (`author`),
   CONSTRAINT `author_FK` FOREIGN KEY (`author`) REFERENCES `contributors` (`username`)
-) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4
+) ENGINE=InnoDB AUTO_INCREMENT=6 DEFAULT CHARSET=utf8mb4

```

Comment:
Why AUTO_INCREMENT sequence start with different value than 1?

Reply:
The value shown is the current value of the counter in our database.

Reply:
Let's remove `AUTO_INCREMENT=6` so it starts from 1 when created. : )